.env.sample

```diff
@@ -78,7 +78,14 @@ OMNIAUTH_ENTRAID_TENANT_ID="<retrieve-from-secrets>"
 
 LAA_LANDING_PAGE_TARGET_URL=<retrieve-from-secrets>
 
-ADMIN_OMNIAUTH_ENTRA_CLIENT_ID="<retrieve-from-secrets>"
-ADMIN_OMNIAUTH_ENTRA_CLIENT_SECRET="<retrieve-from-secrets>"
-ADMIN_OMNIAUTH_ENTRA_TENANT_ID="<retrieve-from-secrets>"
-ADMIN_OMNIAUTH_ENTRAID_REDIRECT_URI="https://localhost:3000/auth/admin_entra_id/callback" # default redirect_uri is taken from EntraID unless supplied here, must be regestered with EntraID regardless
+
+# --------------------------------
+# Admin OMNIAUTH EntraID config
```

Comment:
maybe label the other section at line 69 as `provider OMNIAUTH EntraID config` for completeness